qiskit/qobj/qobj.py

```diff
@@ -26,42 +27,78 @@
 """
 
 
-class QobjSchema(BaseSchema):
```

Comment:
I think the `QobjSchema` should continue to exist and contain parameters such as qobj_id, header, etc. `QASMQobjSchema` and `PulseQobjSchema` should inherit from this.